woocommerce/woocommerce-android
# Jetpack CP: jetpack benefits modal

Closes: #5011 


### Description
This PR just adds the Jetpack benefits modal.

### Testing instructions
- You need to use a Jetpack CP site for the test.
- Open the app, and select this site, then open MyStore screen.
- Click on the jetpack benefits bottom banner.
- Confirm that the jetpack benefits modal is displayed.


### Images/gif
##### Phone
| Light | Dark |
| ---- | ---- |
|  ![Screenshot_1634547130](https://user-images.githubusercontent.com/1657201/137703118-ee5d556d-41d9-4df8-8aa2-c4a5e9012d71.png) |  ![Screenshot_1634547165](https://user-images.githubusercontent.com/1657201/137703160-c211136e-a8a8-4cbf-9b87-2dc98c1c7849.png) |
|  ![Screenshot_1634547146](https://user-images.githubusercontent.com/1657201/137703214-628b19cb-a3fc-4f90-bc05-45dcaa1e6baf.png) | ![Screenshot_1634547173](https://user-images.githubusercontent.com/1657201/137703242-86fe0574-729d-4d70-8164-18d937e82740.png) |

##### Tablet
| Light | Dark |
| ---- | ---- |
|  ![Screenshot_1634547278](https://user-images.githubusercontent.com/1657201/137703333-3165496b-cb78-4c41-aa5f-68476cd45a57.png) |  ![Screenshot_1634547293](https://user-images.githubusercontent.com/1657201/137703366-a009df96-ff8b-45b0-824c-42c99494bb87.png) |
